crates/starknet-devnet/src/main.rs

```diff
@@ -26,11 +26,26 @@ mod cli;
 mod initial_balance_wrapper;
 mod ip_addr_wrapper;
 
+const REQUEST_LOG_ENV_VAR: &str = "request";
+const RESPONSE_LOG_ENV_VAR: &str = "response";
+
 /// Configures tracing with default level INFO,
 /// If the environment variable `RUST_LOG` is set, it will be used instead.
+/// Added are two more directives: `request` and `response`. If they are present, then have to be
+/// removed to be able to construct the `EnvFilter`.
```

Comment:
I don't understand this. I understand how later in the code the removal is implemented, but I don't understand why.

Reply:
because EnvFilter recognise `REQUEST` or `RESPONSE` level not as level but as a path in the program and assigns them the `TRACE` level, which then breaks the tracing configuration

Reply:
What do you mean by "path"? So, without the intervention, putting `REQUEST` in `RUST_LOG` would case `TRACE` to be set? Perhaps this should be commented.

Reply:
path is the path to the file from which info! macro is called

Reply:
[910f68f](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/455/commits/910f68f5a208bd31174607cae4bcfc60f6026c8f)